Core/Src/state_machine.c

```diff
@@ -185,7 +186,7 @@ void vStateMachineDirector(void *pv_params)
 			if (cerberus_state.drive == NOT_DRIVING) {
 				/* make sure foot is on break */
 				bool break_state = false;
```

Comment:
ik this isnt part of ur pr but whats this break state stuff?

Reply:
I think FSAE rules where you need foot on brake to enter active.

Reply:
yeah, makes it so that the drive mode doesnt change unless break is pressed, is rules required for FSAE

Reply:
oh wait this is the defunct state machine